.github/workflows/latest_dependency_checker.yml

```diff
@@ -21,14 +21,15 @@ jobs:
       run: |
         python -m virtualenv venv_core
         source venv_core/bin/activate
-        make installdeps
+        python -m pip install ".[test]"
```

Comment:
I don't think we need the test dependencies here. Wouldn't `python -m pip install "."` work as well since we are only checking featuretools and nltk versions?

Reply:
No, we are checking tensorflow too:
- https://github.com/alteryx/nlp_primitives/blob/remove_py/Makefile#L57

Reply:
This is for `make checkdeps` not `make checkdepscomplete` though. And we aren't using the test requirements for anything are we?

Reply:
Yah actually you're right. Removed this:
- https://github.com/alteryx/nlp_primitives/pull/132/commits/04703a6b80c04833cb31fd08cc165be5cd387aea
- https://github.com/alteryx/nlp_primitives/pull/132/commits/2daee07ec25b9fd07c3984211bed2f6d273d4eaf